python/ray/worker.py

```diff
@@ -1846,7 +1848,7 @@ def method(self):
             infinite retries.
         runtime_env (Dict[str, Any]): Specifies the runtime environment for
             this actor or task and its children.  Currently supports the
-            key "conda_env", whose value should be a string which is the
+            key "conda", whose value should be a string which is the
```

Comment:
Should we replace this with  `A runtime environment dictionary (see``runtime_env.py`` for detailed documentation)` as above?